### PR TITLE
[naoqi_driver_py] disable odom with param

### PR DESCRIPTION
**Pull request by Naoki-Kameyama**
disable odometry with param

get arg from launch file

Because I'd like to test visual SLAM.

### CODE REVIEW

File: naoqi_driver_py/nodes/naoqi_joint_states.py [hunk 1]
```diff
@@ -77,6 +77,7 @@ def __init__(self):
 
         # use sensor values or commanded (open-loop) values for joint angles
         self.useJointSensors = rospy.get_param('~use_joint_sensors', True) # (set to False in simulation!)
+        self.enableOdometry = rospy.get_param('~enable_odometry', True)
```
Comment (k-okada) [inline naoqi_driver_py/nodes/naoqi_joint_states.py]: enable_odometry? use_odometry? not sure which is better..

File: naoqi_driver_py/nodes/naoqi_joint_states.py [hunk 2]
```diff
@@ -157,9 +158,17 @@ def run(self):
 
             t = self.torsoOdom.pose.pose.position
             q = self.torsoOdom.pose.pose.orientation
-            self.tf_br.sendTransform((t.x, t.y, t.z), (q.x, q.y, q.z, q.w),
+          
+
+
+            #if self.odom_param = rospy.get_param('~odom_param', true)
+            if self.enableOdometry:
```
Comment (k-okada) [inline naoqi_driver_py/nodes/naoqi_joint_states.py]: put `t` and `q` within if statement

File: naoqi_driver_py/nodes/naoqi_joint_states.py [hunk 3]
```diff
@@ -157,9 +158,17 @@ def run(self):
 
             t = self.torsoOdom.pose.pose.position
             q = self.torsoOdom.pose.pose.orientation
-            self.tf_br.sendTransform((t.x, t.y, t.z), (q.x, q.y, q.z, q.w),
+          
+
+
+            #if self.odom_param = rospy.get_param('~odom_param', true)
```
Comment (k-okada) [inline naoqi_driver_py/nodes/naoqi_joint_states.py]: is this debug code, may be better to remove, also better to remove blank lines

**Comment by Naoki-Kameyama** [comment]
Thank you for your comment.
I fixed my PR.

**Comment by k-okada** [comment]
lgtm, waiting for another +1 for few days

**Comment by yoneken** [comment]
+1
This PR helps me!

**Comment by k-okada** [comment]
thank you , committed with modified version #59

**Pull request closed without merging**